### PR TITLE
Add Link to profile pic

### LINKED ISSUE #1566: Profile picture on comments messages being not clickable
Right now, the commenter's username is clickable, but not the profile picture:

image

This is a sort of inconsistency, because in profile comments the profile picture **is** clickable.

Should it be clickable or not?

### PR DESCRIPTION
**Pull request by GreenBayRules**
Fixes #1566

### CODE REVIEW

**Comment by GreenBayRules** [comment]
Why did the Travic CI build fail?

**Comment by VutonDesign** [comment]
@GreenBayRules the test came back with 1 problem:
> 137:19 - **error**  Strings must use singlequote  quotes

**Comment by GreenBayRules** [comment]
okay thanks, ill fix it

**Comment by GreenBayRules** [comment]
Approved @chrisgarrity should be okay to merge now right?

File: src/views/messages/message-rows/comment-message.jsx [hunk 1]
```diff
@@ -99,20 +99,21 @@ var CommentMessage = injectIntl(React.createClass({
...
             // must be a project comment, since it's not the other two, and the strict prop type reqs
             if (typeof commentee !== 'undefined' && commentee === this.props.user.username) {
                 return <FormattedMessage
                     id='messages.commentReply'
                     values={{
                         profileLink: <a
                             href={actorLink}
-                            className="social-messages-profile-link"
-                        >
+                            className="social-messages-profile-link">
```
Comment (Kenny2github) [inline src/views/messages/message-rows/comment-message.jsx]: I don't see the reason for this change... could you keep it as
```
profileLink: <a
    href={actorLink}
    className="social-messages-profile-link"
>
```
please?

File: src/views/messages/message-rows/comment-message.jsx [hunk 2]
```diff
@@ -99,20 +99,21 @@ var CommentMessage = injectIntl(React.createClass({
...
                             {this.props.actorUsername}
                         </a>,
                         commentLink: <a href={projectLink}>{this.props.objectTitle}</a>
                     }}
-                />;
+
+                           />;
```
Comment (Kenny2github) [inline src/views/messages/message-rows/comment-message.jsx]: Ditto for here - no reason for change

File: src/views/messages/message-rows/comment-message.jsx [hunk 3]
```diff
@@ -147,11 +149,13 @@ var CommentMessage = injectIntl(React.createClass({
             >
                 <p className="comment-message-info">{messageText}</p>
                 <FlexRow className="mod-comment-message">
+                <a href={url}>
                     <img
                         className="comment-message-info-img"
                         src={commentorAvatar}
                         alt={commentorAvatarAlt}
                     />
+                    </a>
```
Comment (Kenny2github) [inline src/views/messages/message-rows/comment-message.jsx]: Could you fix the indentation here? (Starting from FlexRow:)
```
<FlexRow className="mod-comment-message">
    <a href={url}>
        <img
            className="comment-message-info-img"
            src={commentorAvatar}
            alt={commentorAvatarAlt}
        />
    </a>
    <Comment
...
```

File: src/views/messages/message-rows/comment-message.jsx [hunk 4]
```diff
@@ -99,20 +99,21 @@ var CommentMessage = injectIntl(React.createClass({
             }
         } else {
             var projectLink = '/projects/' + this.props.objectId + '/#comments-' + this.props.commentId;
+
```
Comment (Kenny2github) [inline src/views/messages/message-rows/comment-message.jsx]: what?

**Comment by GreenBayRules** [comment]
Sorry, my editor messed up the indentation and added those weird spaces @Kenny2github

**Comment by GreenBayRules** [comment]
I removed all of those.

File: src/views/messages/message-rows/comment-message.jsx [hunk 5]
```diff
@@ -112,7 +112,7 @@ var CommentMessage = injectIntl(React.createClass({
                         </a>,
                         commentLink: <a href={projectLink}>{this.props.objectTitle}</a>
                     }}
-                />;
+                 />;
```
Comment (Kenny2github) [inline src/views/messages/message-rows/comment-message.jsx]: Weird space?

File: src/views/messages/message-rows/comment-message.jsx (hunk 3 above)
Comment (Kenny2github) [reply src/views/messages/message-rows/comment-message.jsx]: This still stands

**Comment by GreenBayRules** [comment]
I think I fixed it :D @Kenny2github

**Comment by GreenBayRules** [comment]
okay

**Comment by chrisgarrity** [comment]
Thanks to all of you! Nice job @Kenny2github @VutonDesign catching all the little nit-picky things, and thank you @GreenBayRules for making the changes.

**Comment by GreenBayRules** [comment]
Its october xD

**Comment by TheGrits** [comment]
@GreenBayRules @VutonDesign @Kenny2github Milestones start in the middle of each month - just be patient with stuff like pull requests. :)

**Comment by thisandagain** [comment]
The date listed is when we "freeze" the code (that means that all PRs that will be considered for inclusion are merged). Following this we do a week or two of testing to make sure everything is ok before merging all of the changes into the `master` branch. This PR will be reviewed and merged before the date listed (October 18th) and then tested and released before the end of the month if everything checks out ok.

**Comment by GreenBayRules** [comment]
Oh ok, that makes sense.

**Comment by Kenny2github** [comment]
@mewtaylor changes still not available on Production (of course); anything on Staging?

**Comment by mewtaylor** [comment]
On Chrome 61, OSX 10.12 on staging:
* On messages page, I can click a user's avatar and it takes me to their profile
* On messages page, I can click a user's username and it takes me to their profile
Marking this as `qa-ok`

**Pull request merged**